### PR TITLE
feat: Make patch bundles list scrollable

### PR DESCRIPTION
**Pull request by somnisomni**
Column list of patch bundles is now scrollable by this changes.

### CODE REVIEW

File: app/src/main/java/app/revanced/manager/ui/screen/BundleListScreen.kt [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+@Composable
+fun BundleListScreen(
+    onDelete: (PatchBundleSource) -> Unit,
+    onUpdate: (PatchBundleSource) -> Unit,
+    sources: List<PatchBundleSource>,
+    selectedSources: SnapshotStateList<PatchBundleSource>,
+    bundlesSelectable: Boolean = false,
+) {
+    Column {
```
Comment (Ushie) [inline app/src/main/java/app/revanced/manager/ui/screen/BundleListScreen.kt:23]: This column can be removed

File: app/src/main/java/app/revanced/manager/ui/screen/BundleListScreen.kt [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+import app.revanced.manager.ui.component.LazyColumnWithScrollbar
+import app.revanced.manager.ui.component.bundle.BundleItem
+
+@Composable
+fun BundleListScreen(
+    onDelete: (PatchBundleSource) -> Unit,
+    onUpdate: (PatchBundleSource) -> Unit,
+    sources: List<PatchBundleSource>,
+    selectedSources: SnapshotStateList<PatchBundleSource>,
+    bundlesSelectable: Boolean = false,
```
Comment (Ushie) [inline app/src/main/java/app/revanced/manager/ui/screen/BundleListScreen.kt:21]: This default value isn't needed

**Review by Ushie — CHANGES_REQUESTED** [review]

File: app/src/main/java/app/revanced/manager/ui/screen/DashboardScreen.kt [hunk 3]
```diff
@@ -264,33 +263,17 @@ fun DashboardScreen(
...
-                                            if (bundleIsNotSelected) {
-                                                vm.selectedSources.add(it)
-                                            } else {
-                                                vm.selectedSources.remove(it)
-                                            }
-                                        }
-                                    )
-                                }
-                            }
+                            BundleListScreen(
```
Comment (Ushie) [inline app/src/main/java/app/revanced/manager/ui/screen/DashboardScreen.kt:266]: Is this imported correctly? I've not checked in IDE but it looks like there are no imports for the screen

Comment (somnisomni) [reply app/src/main/java/app/revanced/manager/ui/screen/DashboardScreen.kt:266]: I checked this compiles successfully and runs without any issues on AVD when I leave this PR, but maybe I can check this one more time

Comment (somnisomni) [reply app/src/main/java/app/revanced/manager/ui/screen/DashboardScreen.kt:266]: I just checked for it, clean and rebuild project, and launch on AVD, seems no problem. Note that `DashboardScreen` and newly created `BundleListScreen` is in the same package `app.revanced.manager.ui.screen`, so theoretically should have no problem even without the `import` statement.

**Comment by Ushie** [comment]
Would you be interested in being assigned to some issues? Here and possibly on the ReVanced Website repo?

**Comment by somnisomni** [comment]
> Would you be interested in being assigned to some issues? Here and possibly on the ReVanced Website repo?

I don't know if I understood what you said correctly, but as far as I understood, you want me to contribute in other issues on Manager repo and possibly Website repo, right? Sorry for my bad English.

If it's correct, I think I'm not very helpful for it... sorry.

**Pull request merged**